packages/api/src/models/productModel.js

```diff
@@ -36,7 +36,7 @@ class ProductModel extends baseModel {
         product_id: { type: 'integer' },
         name: { type: 'string' },
         product_translation_key: { type: 'string' },
-        supplier: { type: 'string' },
+        supplier: { type: 'string', minLength: 1, maxLength: 255 },
```

Comment:
I get `"error":{"message":"supplier: must NOT have fewer than 1 characters"}` for a farm not pursuing a certification!

Reply:
Oooh thank you for spotting that @SayakaOno!!

I added `null` to the types but noticed that doesn't stop triggering the database error, so removed `minLength` too. I wonder why the other two fields work with just including `null`! 🤔🤔🤔

Reply:
@kathyavini I think we check and trim string on backend for product so an empty string is maybe supplied. (Or to handle api case). I can't see where the model is checked before `checkProductValidity()` though.